### DISCUSSION TITLE
Second Party access to App granted by Third Party

### DISCUSSION THREAD

**Discussion by ClubRun** [vpnhood/VpnHood#443] [General]
Hi GitHub, I'm working on an App idea that involves a model that requires a second party to allow full access to an App by a third party... can this be enabled/arranged? And, the third party pays for the second party full access.      I'll try to explain the scenario - a published App has some basic functions that are useful, but to access the full App functionality a User would nominate a group Administrator (like a Club) to authorise the User's full access. The Administrator would first confirm, for instance, that the User was a member of the "Club" before granting the User full access to the App. There can be multiple Administrators/Groups. Key to this, is the Administrator pays for the App subscription for each User they grant access... so the User doesn't pay for the full access, but they only get full access if the Administrator grants it. I have no idea if this model is possible, but I thought I'd start by asking you...! Thanks for reading this far   Not strictly a coding question I suppose. Maybe you could point me to a more appropriate forum that could help. Thank you GitHub

**Comment by trudyhood** [comment]
Hi. Sorry i believe you posted your idea on a wrong discussion board.

**Reply by ClubRun** [reply]
oh ok. Thanks for that... could you redirct me to the right one...if there is one? thanks again

**Reply by trudyhood** [reply]
Sorry. I don't know
